WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -222,6 +222,12 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required]InputsReque
 						// Check if unconfirmed.
 						if (getTxOutResponse.Confirmations <= 0)
 						{
+							// Check if immature.
+							if (getTxOutResponse.IsCoinBase)
+							{
+								return BadRequest("Provided input is immature.");
+							}
+
```

Comment:
This makes no sense. There isn't such a thing as unconfirmed coinbase.

```suggestion
```

Reply:
I was thinking about before and realized it doesn't make sense.